src/hal/src/format.rs

```diff
@@ -2,7 +2,11 @@
 //! Applicable to textures, views, and vertex buffers.
 
 bitflags!(
-    ///
+    /// Bitflags that describes what properties of a buffer
```

Comment:
"buffer" -> "image"

Reply:
But these properties are applicable to vertex buffers as well, as per the module docs?

Reply:
Vertex buffers can't be COLOR/DEPTH/STENCIL, which is what `AspectFlags` is about